### DISCUSSION TITLE
How can I reducing schedule time ?

### DISCUSSION THREAD

**Discussion by betaincao** [apache/airflow#12579] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/26269330/100039200-bafea500-2e3f-11eb-8349-f4034206093e.png)
As shown in FIG, I want to reduce the schedule time.
I set up max_threads=8，scheduler_heartbeat_sec=5,
Then, how to do reduce the wait time?

**Comment by potiuk** [comment, marked as answer]
In the upcoming Airflow 2.0 (see #10176) soon this time will be significantly reduced. we have beta3 released. Watch for Release Candidate.

**Reply by leahecole** [reply]
Marking this as answered because Airflow 2.0 is released and is definitely our recommended way to improve schedule time! 😄
